tools-core/build.gradle.kts

```diff
@@ -0,0 +1,40 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+plugins {
+    id("com.hedera.hashgraph.java-conventions")
+    id("org.openjfx.javafxplugin") version "0.0.13"
+}
+
+dependencies {
+    implementation(libs.bundles.logging)
+    implementation(libs.bundles.bouncycastle)
+    implementation(libs.jackson.databind)
+    implementation(libs.eddsa)
+    implementation(libs.protobuf.javalite)
+    implementation(libs.protobuf.java.util)
+    implementation(libs.commons.codec)
+    implementation(libs.commons.io)
+    implementation(libs.httpclient)
+    implementation(libs.zt.zip)
+    implementation(libs.opencsv)
+    implementation(libs.picocli)
+//    implementation(libs.json)
+    implementation(libs.grpc.netty)
+    implementation(libs.maven.artifact)
```

Comment:
A maven dependency?

Reply:
It is used for some form of versioning checks. Until I get to the core code, I thought it would be best to leave it in.